CelesteNet.Client/Entities/GhostNameTag.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Microsoft.Xna.Framework;
+﻿using Celeste.Pico8;
```

Comment:
Changes in this file are superfluous? The `SceneAs` vs. `is not` seems okay although idk about the `{ return; }` instead of newline no-braces ;)